wiki/Beatmap_Editor/Timing/en.md

```diff
@@ -1,41 +1,32 @@
-Timing
-==========
+# Timing
 
 ![Timing menu](img/Timing_base.jpg "Timing menu")
 
- The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and beats per minute (BPM). The song's BPM is essentially how fast the song is. The offset is the amount of time between the start of an mp3 and the first downbeat in the song.
+The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and BPM (beats per minute).
+The song's offset is the amount of time between the start of an audio file and the first downbeat in the song.
+The song's BPM is essentially how fast a song is.
 
-The point of having correct timing is so that the timeline in the [editor](/wiki/Beatmap_Editor) lines up with the music. If the timing is incorrect, then the timeline does not match up with the song. Therefore, you cannot accurately place hit objects, and it becomes impossible to map correctly to the music.
+The primary reason to use correct timing is that the [timeline](/wiki/timeline) in the [editor](/wiki/editor) lines up with the music.
```

Comment:
"Correct timing is imperative in the beatmapping process to allow placing [objects](/wiki/hit_objects) in time with the beat."

remove following two sentences, unnecessary.